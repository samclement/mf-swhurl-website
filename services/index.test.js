import { getApiUrl, getMessage } from './'

describe('getApiUrl', () => {
  it('Should get local API URL', () => {
    expect(getApiUrl(true)).toMatchSnapshot()
  })
  it('Should get hostname API URL', () => {
    expect(getApiUrl(false)).toMatchSnapshot()
  })
  it('Should get local API URL by default', () => {
    expect(getApiUrl()).toMatchSnapshot()
  })
})

describe('getMessage', () => {
  it('Should get remote message', async () => {
    expect(await getMessage()).toMatchSnapshot()
  })
})
