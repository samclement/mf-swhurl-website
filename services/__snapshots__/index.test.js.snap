// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getApiUrl Should get hostname API URL 1`] = `"//localhost:"`;

exports[`getApiUrl Should get local API URL 1`] = `"http://localhost:3000"`;

exports[`getApiUrl Should get local API URL by default 1`] = `"http://localhost:3000"`;

exports[`getMessage Should get remote message 1`] = `
Object {
  "completed": false,
  "id": 1,
  "title": "delectus aut autem",
  "userId": 1,
}
`;
